startup.bat

```diff
@@ -1,2 +1 @@
-npm install
```

Comment:
I've separated it into two batch scripts. If you try to run `npm install` offline, it hangs for a while and then craps out